CHANGES.rst

```diff
@@ -13,6 +13,33 @@ Changelog
 
 .. towncrier release notes start
 
+3.24.0 (2023-11-03)
+===================
+
+
+Features
+--------
+
+- Added pulpcore 3.40 compatibility.
+  `#40 <https://github.com/pulp/pulp_rpm/issues/40>`__
```

Comment:
This line ("#40 ...") is a bad apple. Something went wrong in towncrier. Please remove it on the release branch first.

Reply:
https://github.com/twisted/towncrier/issues/562